cpp/_matrix.hpp

```diff
@@ -0,0 +1,55 @@
+#ifndef __MATRIX_HPP__
+#define __MATRIX_HPP__
+
+#include <iostream>
+#include <vector>
+#include <cmath>
+// #include <mkl.h>
+#include <utility>
+#include <iomanip>
+
+// #include <pybind11/pybind11.h>
+// #include <pybind11/stl.h>
+
+class Matrix {
```

Comment:
It will be educating to design a better matrix class.

Reply:
It seems that I can write my Matrix class more effectively. Isn't it?

Reply:
In future PRs, yes.